worker/src/index.ts

```diff
@@ -9,8 +8,6 @@ declare global {
 addEventListener("fetch", (event: FetchEvent) => {
   if (event.request.method === "POST") {
     event.respondWith(handlePost(event.request));
-  } else if (event.request.method === "GET") {
-    event.respondWith(handleGet(event.request));
```

Comment:
We should respond with a 404 probably?

Reply:
Yeah, should od that for everything, not just get

Reply:
https://github.com/home-assistant/analytics.home-assistant.io/pull/73